lightly/embedding/embedding.py

```diff
@@ -56,33 +58,41 @@ class SelfSupervisedEmbedding(BaseEmbedding):
 
     """
 
-    def __init__(self,
-                 model: torch.nn.Module,
-                 criterion: torch.nn.Module,
-                 optimizer: torch.optim.Optimizer,
-                 dataloader: torch.utils.data.DataLoader,
-                 scheduler=None):
+    def __init__(
+        self,
+        model: torch.nn.Module,
+        criterion: torch.nn.Module,
+        optimizer: torch.optim.Optimizer,
+        dataloader: torch.utils.data.DataLoader,
+        scheduler=None,
+    ):
 
         super(SelfSupervisedEmbedding, self).__init__(
-            model, criterion, optimizer, dataloader, scheduler)
+            model, criterion, optimizer, dataloader, scheduler
+        )
 
-    def embed(self,
-              dataloader: torch.utils.data.DataLoader,
-              device: torch.device = None,
-              to_numpy: bool = True):
+    def embed(
+        self, dataloader: torch.utils.data.DataLoader, device: torch.device = None
```

Comment:
Let's do function definitions as outlined in the [style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods): One argument a line, and with typehints:
```python
def embed(self,
          dataloader: torch.utils.DataLoader,
          device: torch.device = None
) -> Tuple[np.ndarray, np.ndarray, List[str]]:
    """DOCSTRING

    """
```

![grafik](https://user-images.githubusercontent.com/65946090/149810977-f92a5dd8-75c5-4d0a-9237-70eb36a1c84b.png)